### PR TITLE
Added Exercise.004/5 and related solutions

### PR DESCRIPTION
**Pull request by mbrigida**

### CODE REVIEW

File: MMUL-BASH-01/Exercises/Day-3/Exercise.004.solutions [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+2) Touch the file, append text and change the mode as requested:
+
+[mmul@machine ~]$ touch user_info.sh
+[mmul@machine ~]$ echo "#!/bin/bash" > user_info.sh
+[mmul@machine ~]$ chmod +x user_info.sh
+[mmul@machine ~]$ ls -l user_info.sh
+-rwxrwxr-x. 1 mmul mmul 12 Dec 11 12:00 user_info.sh
+
+3) The script can be written like that:
```
Comment (rascasoft) [inline MMUL-BASH-01/Exercises/Day-3/Exercise.004.solutions]: Like "this" or "this way"

File: MMUL-BASH-01/Exercises/Day-3/Exercise.004 [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Exercise 004 - Using string variables
+
+1) Log into the machine with the credentials you own;
+
+2) Touch a file called 'user_info.sh', append a bash shebang to it and make it
```
Comment (rascasoft) [inline MMUL-BASH-01/Exercises/Day-3/Exercise.004]: Maybe "append the interpreter (shebang) at the beginning" is better

File: MMUL-BASH-01/Exercises/Day-3/Exercise.005 [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Exercise 005 - Modular scripts with subshells
+
+1) Log into the machine with the credentials you own;
+
+2) Duplicate the script from previous exercise, rename in "user_info_subshell.sh"
```
Comment (rascasoft) [inline MMUL-BASH-01/Exercises/Day-3/Exercise.005]: "rename it in"

**Review by rascasoft — CHANGES_REQUESTED** [review]

File: MMUL-BASH-01/Exercises/Day-3/Exercise.005 (hunk 3 above)
Comment (mbrigida) [reply MMUL-BASH-01/Exercises/Day-3/Exercise.005]: fixed

File: MMUL-BASH-01/Exercises/Day-3/Exercise.004 (hunk 2 above)
Comment (mbrigida) [reply MMUL-BASH-01/Exercises/Day-3/Exercise.004]: fixed, shebang in parenthesis

File: MMUL-BASH-01/Exercises/Day-3/Exercise.004.solutions (hunk 1 above)
Comment (mbrigida) [reply MMUL-BASH-01/Exercises/Day-3/Exercise.004.solutions]: fixed in "like this"

**Pull request merged**